Cargo.toml

```diff
@@ -1,8 +1,18 @@
 [package]
-name = "crate-template"
+name = "vfio-ioctls"
 version = "0.1.0"
-authors = [TODO]
+authors = ["The Cloud Hypervisor Authors", "Liu Jiang <gerry@linux.alibaba.com>"]
+description = "Safe wrappers over VFIO ioctls"
+keywords = ["vfio"]
 license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
 
 [dependencies]
+byteorder = "~1.2.1"
+log = "0.4"
+kvm-bindings = "~0"
+kvm-ioctls = "~0"
+vfio-bindings = "~0"
+vmm-sys-util = "~0"
+
+vm-memory = { git = "https://github.com/rust-vmm/vm-memory", features = ["backend-mmap"] }
```

Comment:
Since vm-memory si now published, can we pull the dependency from crates.io?

Also, why does this crate need to depend on backend-mmap?

Reply:
Moved to vm-memory 0.3.
We want the mmap backend because we need mmap'ed memory regions for mapping the address space into the IOMMU table.